tools/trap/HowToUseTrap.md

```diff
@@ -52,9 +52,11 @@ Device Drivers -> Block-to-character(BCH) Support
 TRAP Script provides two interfaces:  
 1. [CUI](#trap-using-cui)  
 2. [GUI](#trap-using-gui)  
+3. [dutils.sh](#Trap-using-dutils)
```

Comment:
```
There are two ways to execute the TRAP
1. Manual Commands
    1. CUI
    2. GUI
3. Script on Docker - dutils.sh

### Manual Commands
you can use cui or gui and blah blah.....
**NOTE** This requires toolchain installation in your local.

#### CUI

#### GUI

### Script....
You can run the TRAP without toolchain installation in your local. This is working with docker. You can run it by menu selection... blah blah
```
I think dutils and cui / gui is not the same level. Could you list them like above? (Need to rewrite the sentence with proper English)
And please leave a link for each.